MetadataExtractor/Formats/Exif/makernotes/OlympusCameraSettingsMakernoteDescriptor.cs

```diff
@@ -1400,7 +1400,7 @@ private string GetFilterDescription(int tagId)
             for (var i = 0; i < values.Length; i++)
             {
                 if (i == 0)
-                    sb.Append(_filters.ContainsKey(values[i]) ? _filters[values[i]] : "[unknown]");
+                    sb.Append(_filters.TryGetValue(values[i], out string value) ? value : "[unknown]");
```

Comment:
Ditto.

Reply:
Unfortunately, TryGetValue isn't available on .NET3.5. If we drop .NET 3.5, it's a pretty small leap to jump to .NET461.